packages/terra-application-layout/docs/usage.md

```diff
@@ -0,0 +1,278 @@
+# Terra Application Layout
+
+## Prerequisites
+
+- Like all Terra components, the ApplicationLayout requires the presence of a `Base` component (provided by `terra-base`) in its parent hierarchy.
+- Additionally, the ApplicationLayout requires the presence of any `Router` component (provided by `react-router-dom`) in its parent hierarchy.
+
+## Props
+
+### `nameConfig`
+
+The `nameConfig` prop allows consuming applications to add their own branding to the ApplicationLayout. The ApplicationLayout will render this content in various ways based on the current breakpoint.
```

Comment:
nit: current _responsive_ breakpoint. Or maybe _current browser window width_. Not sure if this component has the option to be responsive to parent width or only window width.

Reply:
Sure, I'll clarify.